tests/vmi_lifecycle_test.go

```diff
@@ -430,20 +429,10 @@ var _ = Describe("[rfe_id:273][crit:high][arm64][vendor:cnv-qe@redhat.com][level
 
 					// Creat nonexistent secret, so that the VirtualMachineInstance can recover
 					By("Creating a user-data secret")
-					secret := k8sv1.Secret{
-						ObjectMeta: metav1.ObjectMeta{
-							Name:      "nonexistent",
-							Namespace: createdVMI.Namespace,
-							Labels: map[string]string{
-								util.SecretLabel: "nonexistent",
-							},
-						},
-						Type: "Opaque",
-						Data: map[string][]byte{
-							"userdata": []byte(userData64),
-						},
-					}
-					_, err = kubevirt.Client().CoreV1().Secrets(createdVMI.Namespace).Create(context.Background(), &secret, metav1.CreateOptions{})
+					secret := libsecret.New("nonexistent", map[string][]byte{
```

Comment:
The original code,  set `kubevirt.io/secret` label with some value at line 438:  kubevirt.io/secret=nonexistent.
libesecret.New create a secret with the label with no value: kubevirt.io/secret="", which is fine because the global test suite cleanup doesnt filter by the annotation value (just by its presence)

Reply:
> libesecret.New create a secret with the label with no value: kubevirt.io/secret=""

I see it adds the label with a value of its name, so I still do not understand the sentence and its relevance.

Reply:
My bad, I removed the comment about it from the commit message.

In follow up PR we can revisit how secret are deleted and maybe remove the filtering by the mentioned label.